// react-native-gradient-card-view/build/dist/GradientCard.style.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const react_native_1 = require("react-native");
const { width } = react_native_1.Dimensions.get("window");
function _shadowStyle(shadowColor) {
    return {
        ...react_native_1.Platform.select({
            ios: {
                shadowColor,
                shadowRadius: 5,
                shadowOpacity: 0.5,
                shadowOffset: {
                    width: 1,
                    height: 3,
                },
            },
            android: {
                elevation: 3,
            },
        }),
    };
}
exports._shadowStyle = _shadowStyle;
function _outerContainer(height, width, borderRadius) {
    return {
        width,
        height,
        borderRadius,
    };
}
exports._outerContainer = _outerContainer;
function _imageStyle(height, width, borderRadius) {
    return {
        width,
        height,
        borderRadius,
    };
}
exports._imageStyle = _imageStyle;
exports.default = react_native_1.StyleSheet.create({
    innerContainer: {
        flex: 1,
        marginLeft: 16,
        alignItems: "center",
        flexDirection: "row",
        alignContent: "center",
        alignSelf: "flex-start",
        justifyContent: "center",
    },
    column: { flexDirection: "column" },
    titleStyle: {
        width: 85,
        color: "white",
        marginLeft: 8,
        textAlign: "left",
        fontWeight: "bold",
    },
    subtitleStyle: {
        width: 85,
        fontSize: 12,
        marginLeft: 8,
        color: "#E2E2E2",
        textAlign: "left",
    },
    centerTextContainer: {
        marginLeft: width * 0.01,
    },
    centerTitleStyle: {
        marginLeft: 8,
        color: "white",
        fontWeight: "bold",
        textAlign: "center",
    },
    centerSubtitleStyle: {
        fontSize: 12,
        marginLeft: 8,
        textAlign: "center",
        color: "rgba(163, 224, 97, 1.0)",
    },
    rightContainer: { left: 40 },
});
